code/modules/clothing/masks/boxing.dm

```diff
@@ -18,7 +18,7 @@
 	desc = "Worn by robust fighters, flying high to defeat their foes!"
 	icon_state = "luchag"
 	item_state = "luchag"
-	flags = BLOCKHAIR
+	flags = BLOCKHAIR | MASKCOVERSMOUTH
 	flags_inv = HIDEFACE
 	w_class = 2

```

Comment:
Also, did you HAVE to use a bloody big ass picture to comment with? Now my PR looks silly.